### ISSUE TITLE
How could I put togeter ELU or more advanced activation operation using tensorflow core

### ISSUE THREAD

**Issue by Cr33zz** [SciSharp/TensorFlow.NET#294]
I'm trying to use this library as computational graph only in my neural network implementation. I'm wondering how should I implement functions like ELU or softmax given I have an input Tensor?

**Comment by Oceania2018** [comment]
We already have tf.nn.softmax implemented.

![image](https://user-images.githubusercontent.com/1705364/61593507-e7e4dd00-aba5-11e9-80cf-314325205b05.png)

[tf.nn.elu](https://www.tensorflow.org/api_docs/python/tf/nn/elu) is not ready yet.
